### PR TITLE
New v3 version

### PR DESCRIPTION
**Pull request by Worvast**
#### Added
 * for_logging option in Sender, added more values
 * Logging streamhandler
 * Tests for CLI
 * Have Client query method accept Unix timestamps for to and from dates
 * Make Client parameters retries, timeout, and sleep configurable

#### Changed
 * Errors exceptions in API and Sender are now full controlled and homogenized
 * logging in Common now are created more flexible
 * Sender init class, SSLSender and TCPSender too.
 * API init class
 * Config.from_config to Config.from_dict in API and Sender
 * New documentation
 
#### Fixed
 * Fixed when add key chain to configuration
 * Fixed problems with API CLI Config objects
 * Fixed problems with SENDER CLI Config objects
 * API Proccessors
 * Problems with multiline and zip when both are activated

### CODE REVIEW

File: devo/api/client.py [hunk 1]
```diff
@@ -387,82 +402,86 @@ def _generate_pragmas(self, comment=None):
...
                 try:
                     return json.loads(response.text)
                 except json.decoder.JSONDecodeError:
                     return response.text
             tries += 1
-            time.sleep(self.sleep)
+            time.sleep(self.timeout)
         return {}
 
     def _get_jobs_headers(self):
```
Comment (aarteagamel) [inline devo/api/client.py]: no es valdría igual _get_headers? entiendo que aqui no llegaría el token

File: tests/api/cli.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        configuration = Configuration()
+        configuration.cfg = {"api": {
+            "query": self.query, "address": self.uri,
+            "key": self.key, "secret": self.secret, "token": self.token,
+            "query_id": self.query_id, "user": self.user,
+            "comment": self.comment, "app_name": self.app_name
+        }}
+        self.config_path = "/tmp/devo_api_tests_config.json"
+        configuration.save(path=self.config_path)
+    #
```
Comment (aarteagamel) [inline tests/api/cli.py]: Si son válidos descomenta o si no borra

**Review by aarteagamel — COMMENTED** [review]
E indicar en los readme los cambios de la V2 a la V3

File: README.md [hunk 3]
```diff
@@ -63,13 +63,13 @@ Pull and merge requests are welcome ☺
 ##### Sender
 To send data with Devo SDK, first choose the required endpoint depending on the region your are accessing from:
  * **USA:** 	
-    * **url**: collector-us.devo.io
+    * **url**: usa.elb.relay.logtrust.net
```
Comment (juliovargas) [inline README.md]: aunque `usa.elb.relay.logtrust.net` funciona, pondría `us.elb.relay.logtrust.net` por homogeneidad

File: devo/api/client.py (hunk 1 above)
Comment (Worvast) [reply devo/api/client.py]: Tiene sentido

**Pull request merged**